Sources/KeyboardShortcuts/RecorderCocoa.swift

```diff
@@ -186,7 +186,7 @@ extension KeyboardShortcuts {
 
 					NSAlert.showModal(
 						for: self.window,
-						message: "This keyboard shortcut cannot be used as it's already used by the “\(menuItem.title)” menu item."
+                        message: "This keyboard shortcut cannot be used as it's already used by the".localized + " “\(menuItem.title)” " + "menu item.".localized
```

Comment:
This should be a single string. I think the localization stuff supports interpolation.

Reply:
I have tried a single string, and it seems that Chinese localization does not support interpolation. I do not know why :(

Reply:
I know it's possible, so I would recommend Googling some more ;)